### PR TITLE
Increased code readability through minor logic and style tweaks

### PR DESCRIPTION
**Pull request by sydneysauer**
A few methods in the Satellite class had redundant lines embedded in if/else statements. I simplified these blocks, as well as edited some javadoc comments for minor grammar and style errors.

### CODE REVIEW

File: app/src/main/java/com/neosensory/whatsup/Satellite.java [hunk 1]
```diff
@@ -282,12 +273,11 @@ public String getName() {
...
   public String[] getTles() {
     if (hasTle) {
-      String[] tles = {tle1, tle2};
-      return tles;
+      String[] tles = {tle1, tle2}; 
     } else {
       String[] tles = {"", ""};
-      return tles;
     }
+    return tles;
```
Comment (GreatScott) [inline app/src/main/java/com/neosensory/whatsup/Satellite.java]: Hi @sydneysauer: this modification does not work as the variable `tles` is not defined outside of the if/else structure. A more appropriate modification might be` String[] tles = {"",""} if(hasTle){...` and then get rid of the `else` statement.

**Review by GreatScott — CHANGES_REQUESTED** [review]
Hi @sydneysauer thanks for the PR! Please see my comment about the bug in your modification of `getTles()` in Satellite.java

File: app/src/main/java/com/neosensory/whatsup/Satellite.java (hunk 1 above)
Comment (sydneysauer) [reply app/src/main/java/com/neosensory/whatsup/Satellite.java]: Thanks for catching that...my Java professor would be ashamed of me. :) In that case, the original implementation was simpler than creating one return. I reverted the changes and pushed so it should be OK now.

Comment (sydneysauer) [reply app/src/main/java/com/neosensory/whatsup/Satellite.java]: @GreatScott

**Review by GreatScott — APPROVED** [review]
Tested and works. Thanks for helping to clean up!

**Pull request merged**